File: services/Microservice.MapsPointsUser.Api/Model/Poi.cs
using NetTopologySuite.Geometries;

namespace Microservice.MapsPointsUser.Api.Model
{
    public class Poi
    {
        public Guid PoiId { get; set; }
        public Guid? BusinessId { get; set; }
        public string Name { get; set; }
        public string Category { get; set; }
        public string Description { get; set; }
        public Point Location { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: mobile-android/AndandoAndo_App/AndandoAndo_App/app/src/main/java/com/example/app_andando_ando/domain/model/mapUser/Poi.kt
package com.example.app_andando_ando.domain.model.mapUser

data class Poi(
    val poiId: String,
    val businessId: String?,
    val name: String,
    val category: String?,
    val description: String?,
    val latitude: Double,
    val longitude: Double,
    val imageUrl: String?,
    val menu: List<MenuItem> = emptyList(),
    val openingHours: List<OpeningHour> = emptyList(),
    val averageRating: Double? = null,
    val isActive: Boolean = true,
    val imageLocalPath: String? = null
)

data class MenuItem(
    val menuItemId: String,
    val name: String,
    val description: String?,
    val price: Double
)

data class OpeningHour(
    val dayOfWeek: Int,
    val open: String,
    val close: String
)
